### ISSUE TITLE
Property 'lock' does not exist on type 'ScreenOrientation'.

### ISSUE THREAD

**Issue by iscekic** [microsoft/TypeScript#55539]
### 🔎 Search Terms

N/A

### 🕗 Version & Regression Information

- This changed between versions 5.1.6 and 5.2.2

### ⏯ Playground Link

https://www.typescriptlang.org/play?#code/O4SwdgJg9sB0DOBjATgU1WWVkgwFwEM8QpMAbKRAawAoAiAB2z2QJDwFoGcBbA5AJ50AlAG4gA

### 💻 Code

```ts
window.screen.orientation.lock("portrait-primary");
```

### 🙁 Actual behavior

```
Property 'lock' does not exist on type 'ScreenOrientation'. ts(2339)
```

Which is wrong because it does exist per MDN (and was also working before the version update): https://developer.mozilla.org/en-US/docs/Web/API/ScreenOrientation/lock

### 🙂 Expected behavior

Compiler doesn't complain.

### Additional information about the issue

Interestingly enough, `unlock` is still present on `ScreenOrientation` and doesn't produce this error.

**Comment by MartinJohns** [comment]
Duplicate of https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1615.
